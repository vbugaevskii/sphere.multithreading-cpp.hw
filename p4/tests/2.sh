echo test > file.test
echo test1 test>file.test
cat<file.test 
  cat < file.test   > file1.test
cat file1.test

