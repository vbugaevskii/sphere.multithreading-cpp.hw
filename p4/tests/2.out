test1 test
test1 test
